rockcraft.yaml

```diff
@@ -8,15 +8,6 @@ description: |
     components needed to run PostgreSQL in your Kubernetes
     cluster.
 license: Apache-2.0 # your application's SPDX license
-cmd:
-    - /usr/bin/setpriv
-    - --clear-groups
-    - --reuid
-    - postgres
-    - --regid
-    - postgres
-    - --
-    - /usr/bin/patroni
```

Comment:
We can add a pebble service here, but we don't really need it for the charm.

Reply:
Maybe later, as we want to propose to use rocks separately, e.g: https://github.com/canonical/charmed-mongodb-rock/blob/5-22.04/README.md